virttest/libvirt_xml/base.py

```diff
@@ -320,6 +320,70 @@ def setup_attrs(self, **attrs):
                 target_obj.setup_attrs(**value)
                 setattr(self, key, target_obj)
 
+    def fetch_attrs(self):
```

Comment:
@chloerh 

I notice currently this method is mainly used by  unittests/libvirt_xml/device_xml/test_xx.py. It sounds like it aims to meet test validation requirement. Do you intend to recommend its usage on tp-libvirt? since previously we have some common sense that we get embedded objects first and further get attribute from object as default option when dealing with xml.